src/misc/ErrorsPanel/ErrorsPanel.js

```diff
@@ -17,50 +17,65 @@ import useStyles from './style';
 
 export const ErrorsPanel = (props) => {
   const classes = useStyles();
-  const { errors, labels, onClear, buildErrorsCountLabel } = props;
+  const { errors, maxErrorsCount, labels, onClear, buildErrorsCountLabel } = props;
 
   let errorsCountLabel = labels.errorsCountLabel;
   if (buildErrorsCountLabel) {
-    errorsCountLabel = buildErrorsCountLabel(errors.length);
+    errorsCountLabel = buildErrorsCountLabel(errors.length, maxErrorsCount);
   }
 
+  let errorsBounded;
+  if (errors.length > maxErrorsCount) {
+    errorsBounded = errors.slice(0, maxErrorsCount);
+  }
+
+  const getErrors = () => {
```

Comment:
We may need a `useCallback` here to prevent systematic re-render of the component (@ardvi do you confirm ?)

Reply:
Yess ;) it's better in this component to use useCallback / useMemo for prevent unnecessary renders